<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Disk</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Aqui Va un Logo Macizo de New Disk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMp3Folder">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>20</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnSetExe">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Disk Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File Entries:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnDiskPath">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnTagEntries">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Disk Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnDiskName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>disco.dat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Disk Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set .exe:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDiskPath">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>20</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp3 Folder:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetExe">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>20</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnMp3Folder">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnDiskSize">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>204800</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tag Entries:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnFileEntries">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
